srv/PlanToolPath.srv

```diff
@@ -0,0 +1,6 @@
+string config
```

Comment:
It feels like this config could be a real bear to programmatically fill out. Do you think there would be a reasonable way to, for example, use a file_name parameter to load directly on launch if provided, but override with the config if provided? Maybe as a future improvement.